scripts/apps/vocabularies/components/UploadConfigModal.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import {Button} from 'core/ui/components';
+import ng from 'core/services/ng';
+
+import {Modal} from 'core/ui/components/Modal/Modal';
+import {ModalBody} from 'core/ui/components/Modal/ModalBody';
+import {ModalHeader} from 'core/ui/components/Modal/ModalHeader';
+import {ModalFooter} from 'core/ui/components/Modal/ModalFooter';
+
+import {gettext} from 'core/utils';
+import {dataApi} from 'core/helpers/CrudManager';
+import {UploadComplete} from './UploadComplete';
+import {UploadconfigModalInformation} from './UploadConfigModalInformation';
+import {DropZone} from './drop-zone';
+
+interface IProps {
+    closeModal(): void;
+}
+
+interface IState {
+    files: Array<File>;
+}
+
+const RESOURCE = 'upload/config-file';
+
+export function UploadConfig(updateVocabulary) {
+    return class UploadConfigModal extends React.PureComponent<IProps, IState> {
+        constructor(props: IProps) {
+            super(props);
+
+            this.state = {
+                files: [],
+            };
+        }
+
+        onAddFiles(files) {
+            this.setState({files: Array.from(files || [])});
+        }
+
+        uploadFile() {
+            const notify = ng.get('notify');
```

Comment:
Can you import and use `notify` constant from `scripts/core/notify/notify.tsx`? It avoids direct dependency on angular